tc-worker.Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:16.04
+MAINTAINER Jonas Finnemann Jensen <jopsen@gmail.com>
+
+# Install runtime dependencies
+RUN apt-get update -y \
+ && apt-get upgrade -y \
+ && apt-get install -y qemu-system-x86 qemu-utils dnsmasq-base iptables \
+ && apt-get clean -y \
+ && rm -rf /var/lib/apt/lists/ \
+    ;
+
+# Install zstd 1.0.0
+RUN apt-get update -y \
+ && apt-get install -y curl build-essential \
+ && curl -L https://github.com/facebook/zstd/archive/v1.0.0.tar.gz > /tmp/zstd.tar.gz \
+ && tar -xvf /tmp/zstd.tar.gz -C /tmp \
+ && make -C /tmp/zstd-1.0.0/programs install \
+ && rm -rf /tmp/zstd-1.0.0/ /tmp/zstd.tar.gz \
+ && apt-get purge -y curl build-essential \
+ && apt-get auto-remove -y \
```

Comment:
By removing in the same `RUN` step, files aren't present in the layers... This image is actually only 283MB, where as the `tc-worker-env` used for testing is 766MB (it includes build dependencies like gcc, make and golang).